### DISCUSSION TITLE
Gerar NFe

### DISCUSSION THREAD

**Discussion by gilsonlima10** [OCA/l10n-brazil#3267] [Q&A]
após configurar os parametros para a geração da NFE , mando enviar e visualizar a nota: recebo a seguinte mensagem: 
"Não gerado arquivo PDF!" odoo14 atualizado na sexta feira.

**Comment by antoniospneto** [comment]
Pra entender o que tá acontecendo é preciso de mais informações, tem certeza que a configuração tá correta? Foi selecionado o processador de documento erpbrasil.edoc na configuração da empresa? Sabe dizer se o xml da nota tá gerando?

**Comment by gilsonlima10** [comment]
resolvido
